CHANGELOG.md

```diff
@@ -62,6 +62,67 @@ You should have this:
 <p class="nhsuk-u-font-size-36">
 ```
 
+- Updated default `name` attributes for Date input component
+
+The default name attributes for the date input elements now use square brackets around the date part, for instance `dob[day]`, `dob[month]`, `dob[year]`. Previously they used hyphens (`dob-day`). 
+
+The square brackets mean that the date parts will get combined into an object when using the NHS Prototype Kit, like this:
+
+```json
+{
+  "dob": {
+    "day": "13",
+    "month": "12",
+    "year": "1984"
+  }  
+}
+```
+
+This allows the values to be more easily pre-filled, which you can now do by passing an object to the `values` key:
+
+```njk
+{{ dateInput({
+  namePrefix: "dob",
+  fieldset: {
+    legend: {
+      text: "What is your date of birth?"
+    }
+  },
+  values: data.dob
+}) }}  
+```
+
```

Comment:
I'd add a line showing how to use the individual parts. Eg how to access from nunjucks.